scripts/repo_labels.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+#
+# Simplified BSD License https://opensource.org/licenses/BSD-2-Clause)
```

Comment:
If we're going to include this in the repo there's no reason to license it separately IMO.
As the original gist author I'm +1 changing this to the repo license (GPLv3)